### PR TITLE
Feature | PEO-618 | plan overview

### PR DESCRIPTION
**Pull request by kolyanjy**
JIra - https://garage.atlassian.net/browse/PEO-618

### CODE REVIEW

File: intro_to_circleci.md [hunk 1]
```diff
@@ -606,9 +583,14 @@ workflows:
       - run_specs:
           requires:
             - lintering
-
 ```
+## Plan Overview
+
+In the CircleCi have a different plans for accounts.If you don’t have enough machine power (CPU, memory) you can configure them in the tab `setting -> plan overview` and upgrade them to **PERFORMANCE**.
```
Comment (D3N) [inline intro_to_circleci.md]: Sorry, I think your text has grammar errors, would write smth like this:

There are different plans for accounts in CircleCI. If you need more machine power (CPU, memory) you can upgrade your plan to **PERFORMANCE** in the `settings -> plan overview` tab.

File: intro_to_circleci.md [hunk 2]
```diff
@@ -606,9 +583,14 @@ workflows:
...
+But they are **not** free.
```
Comment (D3N) [inline intro_to_circleci.md]: But this plan is **not** free.

File: intro_to_circleci.md [hunk 3]
```diff
@@ -606,9 +583,14 @@ workflows:
...
 
+Different plan`s price you can see [here](https://circleci.com/pricing/usage/)
 
+If you have many projects, as well as many developers who push their code into different branches. Increase productivity with optional containers. Price for The price for additional containers can be viewed [here](https://circleci.com/pricing/).
```
Comment (D3N) [inline intro_to_circleci.md]: Typo. Price for The price.

File: intro_to_circleci.md [hunk 4]
```diff
@@ -606,9 +583,39 @@ workflows:
       - run_specs:
           requires:
             - lintering
+```
+## Plan Overview
 
+There are different plans for accounts in CircleCI. If you need more machine power (CPU, memory) you can upgrade your plan to **PERFORMANCE** in the `settings -> plan overview` tab.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: Fix all discussed comments on Meeting

File: intro_to_circleci.md [hunk 5]
```diff
@@ -434,22 +434,8 @@ jobs:
...
-
-**Note:** If you are building an open source project on macOS, contact billing@circleci.com to enable these additional containers.
-
-### Security
-While open source can be a liberating practice, take care not to liberate sensitive information.
-
-- If your repository is public, your CircleCI project and its build logs are also public. Pay attention to the information you choose to print.
-
-- Environment variables set in the CircleCI application are hidden from the public, these variables will not be shared in [forked PRs](https://circleci.com/docs/2.0/oss/#pass-secrets-to-builds-from-forked-pull-requests) unless explicitly enabled.
+To support the open source community, projects that are public on GitHub or Bitbucket receive three free build containers, for a total of four containers.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: 3 and 4
Private/Orgs?

File: intro_to_circleci.md [hunk 6]
```diff
@@ -606,9 +583,79 @@ workflows:
...
+      - autoscaling
+      - access to use `resource_class`
+
+    cons:
+      - **High** price (15$ for each 25000 used creedits, 15$ for each user in the account)
+
 
+
+There are different plans for accounts in CircleCI. If you need more machine power (CPU, memory) you can upgrade your plan to **PERFORMANCE** in the `settings -> plan overview` tab.
+But they is **not** free.
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: they?

File: intro_to_circleci.md [hunk 7]
```diff
@@ -606,9 +583,79 @@ workflows:
...
+      - **Unlimited** minutes of use, users and the number of projects 
+
+    cons:
+      - **No** autoscaling
+      - **No** access to use `resource_class`   
+
+2) Transition to a **Performance plan** <br>
+    pros: 
+      - autoscaling
+      - access to use `resource_class`
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: restruct

File: intro_to_circleci.md [hunk 8]
```diff
@@ -606,9 +583,47 @@ workflows:
...
 ```
+## Plan Overview
+Reasons for changing the plan:
+
+- long queues for tests
+- long test time
+
+You can solve these problems in two ways:
 
+1) Purchase of additional containers (each additional container costs $ 50).<br>
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: $50

File: intro_to_circleci.md [hunk 9]
```diff
@@ -606,9 +583,47 @@ workflows:
...
+      - **No** autoscaling
+      - **No** access to use `resource_class`   
+
+2) Transition to a **Performance plan** <br>
+    pros: 
+      - autoscaling
+      - access to use `resource_class`
+
+    cons:
+      - **High** price (15$ for each 25000 used creedits, 15$ for each user in the account)
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: creedits

File: intro_to_circleci.md [hunk 10]
```diff
@@ -606,9 +583,47 @@ workflows:
...
+
+`resource_class` allows you to use different machine instances for change CPU/memory
+
+for change resource plan user **resource_class**. example:
```
Comment (dzemlianoi-double) [inline intro_to_circleci.md]: Use **resource_class** for change resource plan.
Example:

**Pull request merged**